src/test/java/Page/LoginPage.java

```diff
@@ -0,0 +1,46 @@
+package Page;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces. 

For example

`page`